Fix duplicated modal while removing all existing keys in recovery Ledger setup

marcinbodnar opened the pull request.

Patrick1904 commented:
LGTM. Seems we're just missing translated string here:
![Screen Shot 2020-09-18 at 5 52 51 PM](https://user-images.githubusercontent.com/24921205/93655433-180d5300-f9d8-11ea-9c5f-025553a1b79d.png)

marcinbodnar commented:
@Patrick1904 
Added translation. 
Also, added one additional change, because there are at least three actions needed on Ledger while removing keys, so now it will properly show actions needed (instead of showing `processing...` all the time)
